app/models/events/move_cancel.rb

```diff
@@ -0,0 +1,24 @@
+class Events::MoveCancel < GenericEvent
```

Comment:
what do you think about the hierarchy:
```

class GenericEvent
  class MoveCancel < GenericEvent
```
and `generic_event/move_cancel.rb `
this way there is no confusion around previous events and the STI for generic events?

Reply:
Not got strong feelings. Does feel weird using a class as a namespace _and_ for inheritance, though.

Reply:
I did think so too, but found this pattern common with STIs, especially if you already defined `events` as a module somewhere else (I used it in framework response as I had a services module in that name). Not a blocker, however if we go with the current structure of `events` maybe rename to `generic_events` to avoid confusion with the old model.